packages/gitlab/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.4.1"
+  changes:
+    - description: Fix nested key-value pair fields for api and production data streams.
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/12286
```

```suggestion
- version: "2.0.0"
  changes:
    - description: Add mapping for objects values in params field for api and production data streams.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/12286
```

I just want to point out to users that this could be breaking by bumping the major version. Feel free to change the description if you consider that another one is better suited.